### ISSUE TITLE
May I ask how the path is set and how to download the files in the path.

### ISSUE THREAD

**Issue by RecruitXin** [bobo0303/LIGHTWEIGHT-IMAGE-INPAINTING-BY-STRIPE-WINDOW-TRANSFORMER-WITH-JOINT-ATTENTION-TO-CNN#1]
# start from where, the default is latest
resume_ckpt: 'C:/Users/wiwiw/subject20220725/ckpt/20221128_loop_HSV_Stylerewrite_lrdowm5to6_/model_last.pth' #'C:/Users/wiwiw/subject20220725/ckpt/places2_20000_GTLC_202200919_HSVremake_midlayers_morelayer_HSV_wo_V/model_last.pth'
resume_D_ckpt: 'C:/Users/wiwiw/subject20220725/ckpt/20221128_loop_HSV_Stylerewrite_lrdowm5to6_/Discriminator_last.pth' #'C:/Users/wiwiw/subject20220725/ckpt/places2_20000_GTLC_202200919_HSVremake_midlayers_morelayer_HSV_wo_V/Discriminator_last.pth'
test_ckpt: 'C:/Users/wiwiw/subject20220725/ckpt/20221128_loop_HSV_Stylerewrite_lrdowm5to6_/model_bestPSNR.pth' #'C:/Users/wiwiw/subject20220725/ckpt/places2_20000_GTLC_20220831_1704_model_concat_Sig

**Comment by bobo0303** [comment]
resume_ckpt and resume_D_ckpt is for set pre-tain or resume training.
test_ckpt is for load test model.
If you want train this model from head, you need set them None.

We have already set the link at Pretrained model in "readme", but the paper has not been accepted (still feedback stage), so we have turned off the download permission. When our paper is accepted, we'll open it up and we'll update the whole GitHub to make it clearer for users.

Thanks, your review

**Comment by RecruitXin** [comment]
When I set up my resume_ckpt and resume_D_ckpt and test_ckpt paths according to your config file, and start my second training, the system reports&nbsp; errors：


Try load VGG mdoel for Perceptual_loss
D:\Anaconda3-install\install\envs\cnn\lib\site-packages\torch\optim\lr_scheduler.py:129: UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.&nbsp; Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
&nbsp; warnings.warn("Detected call of `lr_scheduler.step()` before `optimizer.step()`. "
D:\Anaconda3-install\install\envs\cnn\lib\site-packages\torch\optim\lr_scheduler.py:508: UserWarning: To get the last learning rate computed by the scheduler, please use `get_last_lr()`.
&nbsp; warnings.warn("To get the last learning rate computed by the scheduler, "

Please tell me how to solve the above problem, thank you




------------------ 原始邮件 ------------------
发件人:                                                                                                                        "bobo0303/LIGHTWEIGHT-IMAGE-INPAINTING-BY-STRIPE-WINDOW-TRANSFORMER-WITH-JOINT-ATTENTION-TO-CNN"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年3月3日(星期五) 中午11:36
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [bobo0303/LIGHTWEIGHT-IMAGE-INPAINTING-BY-STRIPE-WINDOW-TRANSFORMER-WITH-JOINT-ATTENTION-TO-CNN] May I ask how the path is set and how to download the files in the path. (Issue #1)





 
resume_ckpt and resume_D_ckpt is for set pre-tain or resume training.
 test_ckpt is for load test model.
 If you want train this model from head, you need set them None.
 
We have already set the link at Pretrained model in "readme", but the paper has not been accepted (still feedback stage), so we have turned off the download permission. When our paper is accepted, we'll open it up and we'll update the whole GitHub to make it clearer for users.
 
Thanks, your review
